tests/run_tests

```diff
@@ -26,6 +26,7 @@ function testMPI()
 }
 
 testSerial ./SimpleDataTest.cpp.out "Testing simple data read/write..."
+testSerial ./readBool.py "Testing h5py compatible read..."
```

Comment:
yes, the same is true for the `CppUnit` dependency as we got as a feedback from our sys admin.

There are actually some issues for the upcoming [1.3](https://github.com/ComputationalRadiationPhysics/libSplash/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Version+1.3%22) release that will take care of that.